pkger/service_test.go

```diff
@@ -2232,21 +2240,22 @@ func TestService(t *testing.T) {
 									Name: "view name",
 								},
 								Properties: influxdb.BandViewProperties{
-									Type:              influxdb.ViewPropertyTypeBand,
-									Axes:              newAxes(),
-									Geom:              "step",
-									Legend:            influxdb.Legend{Type: "type", Orientation: "horizontal"},
-									Note:              "a note",
-									Queries:           []influxdb.DashboardQuery{newQuery()},
-									HoverDimension:    "y",
-									ShowNoteWhenEmpty: true,
-									ViewColors:        []influxdb.ViewColor{{Type: "text", Hex: "red"}},
-									XColumn:           "x",
-									YColumn:           "y",
-									UpperColumn:       "upper",
-									MainColumn:        "main",
-									LowerColumn:       "lower",
-									TimeFormat:        "",
+									Type:                       influxdb.ViewPropertyTypeBand,
+									Axes:                       newAxes(),
+									Geom:                       "step",
+									Legend:                     influxdb.Legend{Type: "type", Orientation: "horizontal"},
+									Note:                       "a note",
+									Queries:                    []influxdb.DashboardQuery{newQuery()},
+									HoverDimension:             "y",
+									ShowNoteWhenEmpty:          true,
+									ViewColors:                 []influxdb.ViewColor{{Type: "text", Hex: "red"}},
+									XColumn:                    "x",
+									YColumn:                    "y",
+									UpperColumn:                "upper",
+									MainColumn:                 "main",
+									LowerColumn:                "lower",
+									TimeFormat:                 "",
+									LegendOrientationThreshold: 5,
 								},
 							},
 						},
```

Comment:
would it be worthwhile to test values other than 5, just for the sake of showing that it's not implemented to work for 5?

edit: or is 5 meaningful? is it possible to make it more meaningful by assigning it to a variable and naming it?

Reply:
The 5 is meaningless. It's in an assertion that the value should be 5 because the testdata has 5 in the yml and json files. It's similar to what was done with another property, like bin size was arbitrarily chosen as 10 and we assert that we must get 10 for bin size. Same here.